net/znc/Makefile

```diff
@@ -23,6 +23,7 @@ PKG_LICENSE_FILES:=LICENSE
 PKG_USE_MIPS16:=0
 PKG_BUILD_PARALLEL:=1
 
+PKG_CONFIG_DEPENDS := CONFIG_ZNC_ICU
 
```

Comment:
I am obviously not cleaning out the `make menuconfig` properly when testing as I see this menu.